tests/Feature/Controllers/Persons/PersonAddressControllerTest.php

```
@@ -0,0 +1,226 @@
+            ->assertRedirectToRoute('person.show', $person->slug);
+
+        $response->assertSessionHas('status', __('Address created'));
+
+        $address = Address::where('person_id', $person->id)->first();
+        $this->assertFalse($address->is_active);
+    }
+
+    #[Test]
+    public function it_displays_the_edit_address_form(): void
+    {
+        Carbon::setTestNow(Carbon::parse('2025-03-17 10:00:00'));
```

PHPMD (2.15.0)

Avoid using static access to class '\Carbon\Carbon' in method 'it_displays_the_edit_address_form'. (Clean Code Rules)

(StaticAccess)